resources/fastq2sam/fastq2sam.sam

```diff
@@ -0,0 +1,4 @@
+@HD	VN:1.6	SO:queryname
+@RG	ID:A	SM:sample001
+rn001	77	*	0	0	*	*	0	0	GACGGATCTATAGCAAAACT	''))))((*+**+*(**)*+	RG:Z:A
+rn001	141	*	0	0	*	*	0	0	GACGGATCTATAGCAAAACT	''))))((*+**+*(**)*+	RG:Z:A
```

Comment:
this SAM file generated by picard converted the quality scores from the source fastq files:
FFHHHHGGIJIIJIGIIHIJ

into:
''))))((*+**+*(**)*+

@raskoleinonen any idea whats the reason for doing so?